src/main/java/cz/cvut/kbss/termit/service/repository/VocabularyRepositoryService.java

```diff
@@ -140,4 +142,28 @@ public Vocabulary importVocabulary(MultipartFile file) {
     public long getLastModified() {
         return vocabularyDao.getLastModified();
     }
+
+    @Override
+    public void remove(Vocabulary instance) {
+        if (instance instanceof DocumentVocabulary) {
+            throw new VocabularyRemovalException(
+                "Removal of document vocabularies is not supported yet.");
+        }
+
+        final List<Vocabulary> vocabularies = vocabularyDao.getImportingVocabularies(instance);
+        if (!vocabularies.isEmpty()) {
+            List<Vocabulary> vocabularies1 = vocabularies;
```

Comment:
K čemu je toto?

Reply:
To se tam zřejmě dostalo při merge z KBSS TermItu... Přibyl tam i odpovídají REST endpoint atd.

Reply:
Špatně jsem se vyjádřil - komentoval jsem detail, proč zavádíme proměnnou vocabularies1. Ale je to vlastně v metodě, která stejně půjde pryč, tak to klidně mergni.

Reply:
No, dle historie jsi tuhle metodu psal ty, takže se ptáš špatnýho člověka:)